include/openssl/ssl.h

```diff
@@ -1585,12 +1588,13 @@ int SSL_SESSION_up_ref(SSL_SESSION *ses);
 void SSL_SESSION_free(SSL_SESSION *ses);
 __owur int i2d_SSL_SESSION(SSL_SESSION *in, unsigned char **pp);
 __owur int SSL_set_session(SSL *to, SSL_SESSION *session);
```

Comment:
Why leave this one as ```SSL *to```?

Reply:
Less annoying IMHO, it set a `session` into `to` ... clear for me ;)
Feel free to rename it,